File: FIGURAS PYTHON/CIRCLE.py
class Circle:
    pi = 3.14
    def __init__(self, diameter):
        print("creating the circle with diameter: {d}" .format(d=diameter))
        self.radius = diameter/2
    def perimetro(self):
        return 2*self.pi*self.radius
    def Area(self):
        return self.pi*(self.radius*self.radius)
    
    
medium_pizza = Circle(12)
teaching_table= Circle(36)
round_room = Circle(11460)

print("Perimetro")
print(medium_pizza.perimetro())
print(teaching_table.perimetro())
print(round_room.perimetro())

print("Area:")
print(medium_pizza.Area())
print(teaching_table.Area())
print(round_room.Area())

File: FIGURAS JAVA/Circle.java
public class Circle {
    private static final double pi =3.14;
    private double Radius;
    private double Area;
    public Circle(double diameter) {
        System.out.println("creating the Circle with diameter:"+ diameter);
        this.Radius = diameter/2;
    }
    public double Circunference(){return 2*pi*this.Radius;}
    public double Area() { return  pi*this.Radius*this.Radius;}

    public static void main(String[] args) {
        System.out.println("Perimetro:");
        Circle medium_pizza = new Circle(12);
        Circle teaching_table = new Circle(36);
        Circle round_room = new Circle(11400);
        System.out.println(medium_pizza.Circunference());
        System.out.println(teaching_table.Circunference());
        System.out.println(round_room.Circunference());

        System.out.println("Area:");
        System.out.println(medium_pizza.Area());
        System.out.println(teaching_table.Area());
        System.out.println(round_room.Area());
    }
}